packages/editor/src/store/selectors.js

```diff
@@ -861,7 +861,13 @@ export const getEditedPostContent = createRegistrySelector( ( select ) => ( stat
 		postId
 	);
 	if ( record ) {
-		return record.blocks ? serializeBlocks( record.blocks ) : record.content || '';
+		if ( typeof record.content === 'function' ) {
+			return record.content( record );
+		} else if ( record.blocks ) {
```

Comment:
Is this check necessary. I feel like if the edit content is not defined, it probably means we should just return the content.

Reply:
> I feel like if the edit content is not defined, it probably means we should just return the content.

The edit content?

This is the behavior the tests expected.

Reply:
I would have thought `record.content` is the same as serializing ( record.blocks ) if the content is not set as an edit (serialization of blocks). In my mind, this means that the blocks were not edited which also means the content was not edited (persisted value).

Reply:
This also supports editing `blocks` without setting `content` to a function first. (For the first non persistent edits.)

Reply:
> This also supports editing blocks without setting content to a function first. (For the first non persistent edits.)

I don't think setting a non persistent edit without saying how this non-persistent edit affects the persistent edit (content) should change the value of the "edited version of the persisted edit" (aka `getEditedPostContent`).

That said, it seems minor.

Reply:
(Backwards Compatibility)

Reply:
Right that's a good reason.

Reply:
Strange, Github double posted my comment.

Reply:
haha me too

Reply:
Wow, it's changing the timestamps/order now!